src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -87,27 +87,39 @@ def plot_fit(lc, model, meta, fitter, isTitle=True):
 
         residuals = flux - model
 
+        # Get binned data and times
+        if not hasattr(meta, 'nbin_plot') or meta.nbin_plot is None or \
+           meta.nbin_plot > len(lc.time):
+            nbin_plot = len(lc.time)
```

Comment:
These should use `time` and not `lc.time` (because `lc.time` can be longer than `time` for multi-white fits).